doc/manuals/devel/cprs.md

```diff
@@ -16,7 +16,7 @@ In NGSIv1 (deprecated), the request `POST /v1/updateContext` has a field called
 * APPEND_STRICT
 * REPLACE
 
-> Side-node: The first three are "standard NGSIv1" while the second two were added for NGSIv2.
```

Comment:
This file modificaiton seems to be related with fixes in the query forward documentation, but the update forwared documetnation about (FW-02 if I'm remembering correctly) has not be modified. Probably an update of the png for FW-02 is needed also, but I haven't seen yet the changes in the code so I'm not fully sure.

Reply:
As you probably can see, the modifications in this file are just corrections.

About documenting _Update Forward_, part of it is added, but most probably not **all** that is necessary